### ISSUE TITLE
Deep Learning and Structured Prediction for the Segmentation of Mass in Mammograms

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#168]
http://doi.org/10.1007/978-3-319-24553-9_74
> In this paper, we explore the use of deep convolution and deep belief networks as potential functions in structured prediction models for the segmentation of breast masses from mammograms. In particular, the structured prediction models are estimated with loss minimization parameter learning algorithms, representing: a) conditional random field (CRF), and b) structured support vector machine (SSVM). For the CRF model, we use the inference algorithm based on tree re-weighted belief propagation with truncated fitting training, and for the SSVM model the inference is based on graph cuts with maximum margin training. We show empirically the importance of deep learning methods in producing state-of-the-art results for both structured prediction models. In addition, we show that our methods produce results that can be considered the best results to date on DDSM-BCRP and INbreast databases. Finally, we show that the CRF model is significantly faster than SSVM,both in terms of inference and training time, which suggests  an advantage of CRF models when combined with deep learning potential functions.

**Comment by cgreene** [comment]
The challenge here is to identify the locations of masses in mammogram images.

I'm going to focus my discussion here on the 'only' categories of the results (the others become more challenging to interpret & also make comparison with subsequent work more difficult).

There are very few training examples (58+39) and testing examples (58+40). Structure of their neural net approaches is in Figure 2A. The NNs are used as feature constructors, assuming I am reading this correctly.

There may be some concerns around the validation numbers if we want to take them as the ground truth:
> 1) we tried different CNN structures and the combination of more than one CNN model as additional potential functions, but the single CNN model detailed in Sec. 3.1 produced the best cross validation results; 2) for the DBN models, we tried different input sizes (3 × 3 and 7 × 7 patches), but the combinations of the ones detailed in Sec. 3.1 provided the best cross-validation results;

There's enough work in this area that we probably should discuss it in general. I don't know that it provides, at this point, evidence of transformative potential.

**Comment by cgreene** [comment]
relates to #163 #164

**Comment by cgreene** [comment]
discussed in latest commits

**Comment by myouesfi** [comment]
Hi,

Would you let me know, is it possible to access to this paper implementation.
Thanks,

**Comment by agitter** [comment]
Sorry @myouesfi, this repository is a third-party discussion of literature for a review manuscript.  None of us are affiliated with the paper above or have access to the implementation.
